From 083d8c68da682bfa5dfe2b4c8ebe332eddbd8643 Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Fri, 13 Jun 2014 10:41:19 +0800
Subject: [PATCH 0076/1543] ENGR00317981: drm: vivante: drm_fasync is gone
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit a3cc6eb8822029fcd85dba77b168ca40187f65d9 from
git://git.freescale.com/imx/linux-2.6-imx.git

The drm_fasync() is gone since commit b0e898ac555e (drm: remove FASYNC
support).  Remove it from vivante driver to fix the build error below.

  CC      drivers/gpu/drm/vivante/vivante_drv.o
drivers/gpu/drm/vivante/vivante_drv.c:68:12: error: ‘drm_fasync’ undeclared here (not in a function)

Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 drivers/gpu/drm/vivante/vivante_drv.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/vivante/vivante_drv.c b/drivers/gpu/drm/vivante/vivante_drv.c
index 4caf46f..d64dfa8 100644
--- a/drivers/gpu/drm/vivante/vivante_drv.c
+++ b/drivers/gpu/drm/vivante/vivante_drv.c
@@ -65,7 +65,6 @@ static const struct file_operations viv_driver_fops = {
 	.unlocked_ioctl = drm_ioctl,
 	.mmap = drm_mmap,
 	.poll = drm_poll,
-	.fasync = drm_fasync,
 	.llseek = noop_llseek,
 };
 
-- 
1.7.5.4

